README.md

```diff
@@ -9,7 +9,7 @@ This repository contains the source for the [Rundeck](http://rundeck.org/) [dock
 1. Supervisor, Apache2, and rundeck
 1. No SSH.  Use docker exec or [nsenter](https://github.com/jpetazzo/nsenter)
 1. If RUNDECK_PASSWORD is not supplied, it will be randomly generated and shown via stdout.
-1. Supply the SERVER_URL or else you won't get too far :)
+1. Supply the SERVER_URL or it will default to https://0.0.0.0:44443
```

Comment:
@johnseekins Yes, that's right.  I was pointing out that you have an extra 4 in the README update 😛 

```
it will default to https://0.0.0.0:44443
```

Should be
```
it will default to https://0.0.0.0:4443
```